data/js/duckduckgo.js

```diff
@@ -1,4 +1,16 @@
-self.port.emit('loaded');
+// grab the ATB param from the html element's class
+function getATBParam() {
+  var htmlClasslist = document.querySelector("html").classList;
+  for (var i = 0; i < htmlClasslist.length; i++) {
+    var cls = htmlClasslist[i];
+    if (cls.match(/^atb-/)) {
+      return cls.replace(/^atb-/, '');
+    }
+  }
+}
+
+var atbParam = getATBParam();
+self.port.emit('loaded', atbParam);
```

Comment:
@bsstoner using it this way means that the `atbParam` might be `undefined` in some circumstances. Do you think it makes sense to check for that?